### PR TITLE
libggz: update mirror

### PR DESCRIPTION
**Pull request by queengooborg**

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
@BrewTestBot Test this again, please?

**Comment by vinyldarkscratch** [comment]
@BrewTestBot test this please

File: Formula/libggz.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Libggz < Formula
   desc "The libggz library wraps many common low-level functions"
   homepage "http://dev.ggzgamingzone.org/libraries/libggz/"
-  url "http://mirrors.ibiblio.org/ggzgamingzone/ggz/0.0.14.1/libggz-0.0.14.1.tar.gz"
+  url "https://ftp.osuosl.org/pub/blfs/conglomeration/libggz/libggz-0.0.14.1.tar.gz"
```
Comment (ilovezfs) [inline Formula/libggz.rb]: let's note that this is a mirror. I think the original url is blocking non-browser access but is still valid

Comment (vinyldarkscratch) [reply Formula/libggz.rb]: I tried to access the original URL through Chrome, and it couldn't reach the website either.  🙁

Comment (ilovezfs) [reply Formula/libggz.rb]: actually it looks like the other official mirrors are good
See http://ftp.ggzgamingzone.org/

http://mirrors.dotsrc.org/ggzgamingzone/ggz/0.0.14.1/libggz-0.0.14.1.tar.gz
http://ftp.belnet.be/mirror/ftp.ggzgamingzone.org/ggz/0.0.14.1/libggz-0.0.14.1.tar.gz

Comment (vinyldarkscratch) [reply Formula/libggz.rb]: Ah, sweet, so it's just iBiblio that's down, then!

**Comment by ilovezfs** [comment]
Thanks again @vinyldarkscratch! :white_check_mark:

**Pull request merged**